tests/openai/test_openai_autolog.py

```diff
@@ -402,3 +420,37 @@ def test_autolog_raw_response_stream(client):
     assert len(trace.data.spans) == 1
     span = trace.data.spans[0]
     assert span.outputs == "Hello world"
+
+
+@pytest.mark.parametrize("tools", [None, MOCK_TOOLS])
+@pytest.mark.parametrize("stream", [True, False])
+def test_chat_schema_attributes_are_set(client, tools, stream):
```

Comment:
nit: I'd add the attribute check to existing tests rather than create a new test case; Let's say we add async support in the future, then I would imagine those existing test cases are copied or extended. If we have the attribute check outside, it's easy to forget adding that test coverage.